src/java.desktop/windows/native/libjsound/PLATFORM_API_WinOS_DirectSound.cpp

```diff
@@ -479,7 +479,7 @@ DS_StartBufferHelper::Data::~Data() {
 
 DWORD WINAPI __stdcall DS_StartBufferHelper::ThreadProc(void *param)
 {
-    ::CoInitialize(NULL);
+    ::CoInitializeEx(NULL, COINIT_MULTITHREADED);
```

Comment:
The documentation for the `CoInitializeEx` suggests to disable the `COINIT_DISABLE_OLE1DDE`: 
https://learn.microsoft.com/en-us/windows/win32/learnwin32/initializing-the-com-library

I think we do not need ole1 here and can disable that.

Reply:
@mrserb, thank you for your review I have disabled ole1 by adding the flag `COINIT_DISABLE_OLE1DDE`.